iptables/table.go

```diff
@@ -205,6 +208,12 @@ type Table struct {
 	// to what we calculate from chainToContents.
 	chainToDataplaneHashes map[string][]string
 
+	// chainToFullRules contains the full rules, mapped from chain name to slices of rules in that chain.
+	chainToFullRules map[string][]string
+
+	// topLevelChains is the set of top-level chains we insert into.
+	topLevelChains set.Set
```

Comment:
Just remembered `tableToKernelChains` above, I think the way we use that makes this redundant since we pre-populate `chainToInsertedRules` with empty slices for all the top-level chains!

Reply:
Good catch, thanks